XPI/index.js

```diff
@@ -120,8 +120,8 @@ tabs.on('activate', function() {
 });
 
 function getActiveWorker() {
-	let tab = tabs.activeTab;
-	for (let i in workers) {
+	const tab = tabs.activeTab;
+	for (const i in workers) {
```

Comment:
Odd, I see http://stackoverflow.com/questions/31987465/ecmascript-2015-const-in-for-loops too. I'll change it to fix Firefox, but hopefully Firefox fixes their implementation of `const`.